src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/Current_Routine_DropAndCreateRelease.sql

```diff
@@ -10,18 +10,28 @@ BEGIN
     IF LEN(ISNULL(@LinkedFootnotes, '')) > 0
         INSERT @LinkedFootnotesList SELECT value FROM STRING_SPLIT(@LinkedFootnotes,',');
 
+    DECLARE @FootnoteIds TABLE (Id UNIQUEIDENTIFIER PRIMARY KEY NOT NULL);
+    INSERT INTO @FootnoteIds (Id) SELECT RF.FootnoteId FROM dbo.ReleaseFootnote RF WHERE ReleaseId = (SELECT R.Id FROM @Release R);
+
+    DELETE FROM dbo.SubjectFootnote WHERE FootnoteId IN (SELECT F.Id FROM @FootnoteIds F);
```

Comment:
Take it you didn't like my suggestion of deleting with a join? 🙂

Reply:
Just not seen that syntax before so I sidestepped it :)

I've switched to using a join now